flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliTableauResultView.java

```diff
@@ -132,11 +145,23 @@ private void printResults(AtomicInteger receivedRowCount, boolean isStreamingMod
                             DEFAULT_COLUMN_WIDTH,
```

Comment:
If you check the signature of `PrintUtils.columnWidthsByType` and you will see the second parameter is *maxColumnWidth*. DEFAULT_COLUMN_WIDTH will not be enough even for some popular data types such as `TIMESTAMP(3)`

Reply:
reuse `PrintUtils.MAX_COLUMN_WIDTH`

```
+--------------------------+
|        current_timestamp |
+--------------------------+
| 2021-04-12T08:59:24.973Z |
+--------------------------+
Received a total of 1 row
```